project_status/views/project.xml

```diff
@@ -0,0 +1,93 @@
+<odoo>
+  <!-- List View-->
+  <record id="project_status_view_inherit_list" model="ir.ui.view">
+    <field name="name">project.status.tree</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.view_project"/>
+    <field name="type">tree</field>
+    <field name="field_parent">child_ids</field>
+    <field name="arch" type="xml">
+      <field name="user_id" position="after">
+        <field name="project_status"/>
+      </field>
+    </field>
+  </record>
+
+  <!-- Form View -->
+  <record id="project_status_view_inherit_form" model="ir.ui.view">

```

Comment:
Would be nice if you use the original id (optional)

Reply:
I've changed the id to the original's id. Thanks.